src/components/utils/DateUtils.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+export const formatDate = (timestamp: string): string => {
```

Comment:
we do not need this. All time formatting must happen in backend. We just display whatver the API returns.